client/src/main/java/com/paypal/selion/reports/reporter/excelreport/ReportMap.java

```diff
@@ -87,9 +87,13 @@ public void addToGeneratedReport(List<BaseReport<V>> lstCustomReport) {
         for (String indReport : this.reportData.keySet()) {
             if (iTypeOfReport == 0) {
                 b = new SummaryReport(indReport);
-            } else {
```

Comment:
the previous way of formatting the `else` conditions is preferred.  Use `} else {`